assets/sass/modules/navigation/_global-navigation.scss

```diff
@@ -0,0 +1,60 @@
+//--------------------------------------------------------------
+// Global and Shared Navigation Styles
+//--------------------------------------------------------------
+
+// Default menu (active after a fresh WordPress install)
```

Comment:
@jomurgel All single line comments should end with punctuation. I know it's inconsistent throughout this theme, but if you could fix up the comments in this PR, that'd be 👍